bin/generate-manifest.js

```diff
@@ -14,7 +14,7 @@ const manifest = Object.assign(packageMeta.webextensionManifest, {
       'id': packageMeta.id
     })
   }
-});
+}, packageMeta.webextensionManifest);
```

Comment:
So, this will make it so that the name in `webextensionManifest` will override the name we specify here, which makes me wonder why we are still specifying the name up on line 8…

Reply:
Hmm, looks like I was kind of overthinking it in case this script gets used for another add-on and the name *doesn't* need to be overridden. But, npm has all kinds of restrictions on name that are inconvenient for an add-on name that we might as well always specify it